src/components/button/button.scss

```diff
@@ -14,6 +16,13 @@
 }
 
 button {
+    @include is-elevated-clickable();
```

Comment:
is it on purpose that the button has different styles for `:active` and `:focus`?
active/ clicked: 
![image](https://user-images.githubusercontent.com/10260886/86016996-ce8a1900-ba23-11ea-8fb3-a7899c2cf86a.png)
focused but but active (closed alert): 
![image](https://user-images.githubusercontent.com/10260886/86017117-f0839b80-ba23-11ea-88f4-cc01a545c7c5.png)

button needs to lose its focus to turn back to the neutral state.

Reply:
Hmm... I haven't added any styles for `focus` state, unless MD has something. However the styles for `active` state is there to indicate what happens when you press/click the button. It will go down with an animation. To test it, click the button but don't immediately release your click.

Reply:
We will at some point make some unified styles for focused states for all components. Today, we have no good and nice design for them. Already had some thought exchange regarding this with @adrianschmidt

Reply:
> To test it, click the button but don't immediately release your click.

that's fine, but it looks like we capture the same active state, when the button opens the alert (I haven't tested whether it's the same with a regular dialog). Is that on purpose?

Reply:
Yeah, MDC has two different looks for focused and active. The very subtle change of colour between focused and blurred is probably MDC's way to try to balance the need for focus indication for keyboard users with the need for a focus indicator that doesn't look very ugly or weird for mouse users, since the focus will stay on the button after clicking (which is a browser behaviour, that we can't really do much about).

There's a lot of different factors to consider around focus and focus indication, several of which are conflicting, so just let it be like it is for now, and we'll sit down and do some good design work specifically for focus handling on all of our components at some point. If you're interested, you'd be more than welcome to join that effort!

Reply:
Oh, my "yeah" was referring to Kia's replies. I didn't see your reply until now Katja.

If I'm understanding your question correctly, I think the OK-button becomes focused immediately as the dialog opens, right? And yes, that's on purpose, because it lets the user close the dialog by pressing Enter.

Reply:
I just wanted to make sure that this was intentionally and not a bug. If that's the case I'm totally fine with the changes